mathics_django/doc/django_doc.py

```diff
@@ -28,21 +29,25 @@
 from mathics_django.settings import get_doctest_html_data_path
 
 # FIXME: remove globalness
+doctest_html_data_path = get_doctest_html_data_path(should_be_readable=True)
 try:
-    doctest_html_data_path = get_doctest_html_data_path(should_be_readable=True)
     with open(doctest_html_data_path, "rb") as doctest_html_data_file:
         doc_data = pickle.load(doctest_html_data_file)
 except IOError:
     print(f"Trouble reading Doc file {doctest_html_data_path}")
     doc_data = {}
 
 
-class DjangoDocElement(object):
+class DjangoDocElement:
+    """
+    Adds some HTML functions onto existing Django Document Elements.
+    """
+
     def href(self, ajax=False):
```

Comment:
My environment tells me that something is weird with calling `self.get_url()`.  `get_url` is not defined in this class and it does not inherit from anything. Therefore were this to get called, we'd have an error.

Reply:
I couldn't find `get_url` in this file. What is there is `get_uri`, which is defined and used to build the navigation links. `get_title_html` is defined for `DjangoDocElement`, and I think we need it for doing the documentation format.

Reply:
What the linter is saying is that _if_ one tries to use  it, you'll get an error. 

I did some git archeology and this was added by me around 2020 when I split this off from what was in `mathics-core`.  There was a _lot_ of guess work there. 

There is no such thing as "building" documentation for Django. So we can remove this.

Reply:
OK, but both functions seem to work properly, as they are called when navigating documentation. I added some comments in the docstrings to clarify where they are used.